modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -7,110 +7,71 @@
 == Understanding how to encourage the JVM to release unused memory to the operating system
 
-By default, the OpenJDK does not aggressively return unused memory to the
-operating system. This may be appropriate for many containerized Java
-workloads, but notable exceptions include workloads where additional active
-processes co-exist with a JVM within a container, whether those additional
-processes are native, additional JVMs, or a combination of the two.
+By default, OpenJDK does not aggressively return unused memory to the operating system. This may be appropriate for many containerized Java workloads, but notable exceptions include workloads where additional active processes co-exist with a JVM within a container, whether those additional processes are native, additional JVMs, or a combination of the two.
 
-Java-based agents can use the following JVM arguments to encourage the JVM
-to release unused memory to the operating system:
+Java-based workloads can use the following JVM arguments to encourage the JVM to release unused memory to the operating system:
 
 [source,terminal]
 ----
 -XX:+UseParallelGC
 -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:GCTimeRatio=4
--XX:AdaptiveSizePolicyWeight=90.
+-XX:AdaptiveSizePolicyWeight=90
 ----
 
-These arguments are intended to return heap
-memory to the operating system whenever allocated memory exceeds 110% of in-use
-memory (`-XX:MaxHeapFreeRatio`), spending up to 20% of CPU time in the garbage
-collector (`-XX:GCTimeRatio`). At no time will the application heap allocation
-be less than the initial heap allocation (overridden by `-XX:InitialHeapSize` /
-`-Xms`). Detailed additional information is available
-link:https://developers.redhat.com/blog/2014/07/15/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-1/[Tuning Java's footprint in OpenShift (Part 1)],
-link:https://developers.redhat.com/blog/2014/07/22/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-2/[Tuning Java's footprint in OpenShift (Part 2)],
-and at
-link:https://developers.redhat.com/blog/2017/04/04/openjdk-and-containers/[OpenJDK
+These arguments are intended to return heap memory to the operating system whenever allocated memory exceeds 110% of in-use memory (`-XX:MaxHeapFreeRatio`), spending up to 20% of CPU time in the garbage collector (`-XX:GCTimeRatio`). At no time will the application heap allocation be less than the initial heap allocation (overridden by `-XX:InitialHeapSize` /
+`-Xms`). Detailed additional information is available in link:https://developers.redhat.com/blog/2014/07/15/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-1/[Tuning Java's footprint in OpenShift (Part 1)], link:https://developers.redhat.com/blog/2014/07/22/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-2/[Tuning Java's footprint in OpenShift (Part 2)], and at link:https://developers.redhat.com/blog/2017/04/04/openjdk-and-containers/[OpenJDK
 and Containers].
```

Comment:
Probably out of scope for this PR but it would be nice to organize this as a list of arguments and what they do instead of a block that lists them and then a paragraph under - it's a bit hard to follow